src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -1,4 +1,5 @@
-! > Calculates Zanna and Bolton 2020 parameterization
+!> Calculates Zanna and Bolton 2020 parameterization
+!! Implemented by Perezhogin P.A. Contact: pperezhogin@gmail.com
 module MOM_Zanna_Bolton
```

Comment:
Include a license statement?
```

! This file is part of MOM6. See LICENSE.md for the license.

```
You should be able to check other files for examples.